### PR TITLE
Add configurability for DB connection params;

### PR DESCRIPTION
**Pull request by stevejalim**
Starting out with CONN_MAX_AGE of ~600~ 0, to match the default. We can then tune from there

### CODE REVIEW

File: bedrock/settings/base.py [hunk 1]
```diff
@@ -65,11 +65,18 @@ def data_path(*args):
...
+
 DATABASES = {
-    "default": config(
-        "DATABASE_URL",
-        default=f"sqlite:////{data_path('bedrock.db')}",
-        parser=dj_database_url.parse,
+    "default": dj_database_url.parse(
+        db_default_url,
+        conn_max_age=db_connection_max_age_secs,
+        conn_health_checks=db_conn_health_checks,
```
Comment (robhudson) [inline bedrock/settings/base.py:79]: Do these get ignored if sqlite?

Comment (stevejalim) [reply bedrock/settings/base.py:79]: Yeah i've tried it locally and they seem happy
I can push this to bedrock-test to be sure

Comment (stevejalim) [reply bedrock/settings/base.py:79]: https://github.com/mozilla-sre-deploy/deploy-bedrock/actions/runs/9687725130

Comment (robhudson) [reply bedrock/settings/base.py:79]: cool, just didn't know if we needed a "if postgres" check.

**Review by robhudson — APPROVED** [review]

**Comment by stevejalim** [comment]
@robhudson FYI i made the CONN_MAX_AGE to default `0` so that this will be an inert change

**Pull request merged**